### ISSUE TITLE
old PCR Test should vanish

### ISSUE THREAD

**Issue by viisauksena** [corona-warn-app/cwa-wishlist#773]
old PCR Tests are still in Frontpage, even if weeks old. 
If someone not deletes them actively the old PCR Test should move somewhere else, since the app should go in business as usual mode. (maybe ask for it) 
Otherwise the main view is blocked by an very old pcr positive test.
Or give some information on how to get a "genesenen certificate" instead, 
or the info that the pcr test - if you are healthy now - is no longer needed and no good for anything.

---
Internal Tracking ID: [EXPOSUREAPP-11578](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11578)

**Comment by larswmh** [comment]
Thanks for your suggestions @viisauksena, @Jana-Z and @steffmueller. This issue has been moved to the cwa-wishlist repository as it is not a bug. We have created an internal ticket for it and will raise this topic internally. Internal Tracking ID: [EXPOSUREAPP-11578](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11578)

---
Corona-Warn-App Open Source Team

**Comment by HolgerJeromin** [comment]
I still have a "scary" red display of my PCR Test 202 days afterwards. 
This is kind of silly to keep on the homescreen for longer than a month.

If you do not want to delete it, it should be at least be "less red" because the info "you are probably infectious" and "your local government will perhaps contact you in the next days" is bogus.

**Comment by Ein-Tim** [comment]
I also want to stress this point. Negative tests are automatically shown as invalid after a given period of time. The same should be the case for positive tests.

**Comment by brianebeling** [comment]
> I still have a "scary" red display of my PCR Test 202 days afterwards. This is kind of silly to keep on the homescreen for longer than a month.
> 
> If you do not want to delete it, it should be at least be "less red" because the info "you are probably infectious" and "your local government will perhaps contact you in the next days" is bogus.

I generally agree that the test doesn't need to be shown on the home screen after the fact. Is there a specific reason you do not want to delete the test?

**Comment by HolgerJeromin** [comment]
> Is there a specific reason you do not want to delete the test?

1. Curiosity when the test is cleaned up
2. Fear to delete something "important"
3. Having a proof of a positive test in the past (when having long/post covid). But yes the QR code of the recover is much more useful for that :-)

Personally I am in category 1, but the other two are probably found in the wild, too.

**Comment by brianebeling** [comment]
> > Is there a specific reason you do not want to delete the test?
> 
> 1. Curiosity when the test is cleaned up
> 2. Fear to delete something "important"
> 3. Having a proof of a positive test in the past (when having long/post covid). But yes the QR code of the recover is much more useful for that :-)
> 
> Personally I am in category 1, but the other two are probably found in the wild, too.

Thank you very much for the insight, I think those are very valid reasons as to why people do not want to delete the tests or why it feels bad to do so. I added your feedback to the internal ticket.
